tests/test_project.py

```diff
@@ -122,6 +123,70 @@ def test_status_performance(self):
         assert time < 10
 
 
+class TestProjectStatusFilterOperations(TestProjectBase):
+    project_class = define_status_test_project._TestProject
+
+    def mock_project(self):
+        project = self.project_class.get_project(path=self._tmp_dir.name)
+
+        for a in range(2, 4):
+            for b in range(2):
+                job = project.open_job(dict(a=a, b=b)).init()
+                job.doc.a = a
+                job = project.open_job(dict(a=dict(a=a), b=b)).init()
+                job.doc.b = b
+
+        return project
+
+    @pytest.fixture(scope="function")
+    def project(self):
+        return self.mock_project()
+
+    @pytest.fixture(scope="function")
+    def get_status(self, project):
+        def _get_status(kwargs):
```

Comment:
Should this function accept `**kwargs` instead? Then we could refactor the call sites like `get_status(operation=groups)` instead of `get_status({"operation": groups})`. I think this is a good idea because it makes `get_status` have a similar signature to `project.print_status`.
```suggestion
        def _get_status(**kwargs):
```